test/routers/start.int.ts

```diff
@@ -1,39 +1,80 @@
 import * as cheerio from "cheerio";
-import middlewareMocks from "../mocks/allMiddleware.mock";
+import mockSessionMiddleware from "../mocks/sessionMiddleware.mock";
+import mockFetchCompanyMiddleware from "../mocks/fetchCompany.mock";
+import mockFetchVerificationMiddleware from "../mocks/fetchVerification.mock";
 import request from "supertest";
 import app from "../../src/app";
 import { PrefixedUrls, servicePathPrefix } from "../../src/constants";
 import { HttpStatusCode } from "axios";
+import { checkPlannedMaintenance } from "../../src/services/pscVerificationService";
+import { ApiResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
+import { PlannedMaintenance } from "@companieshouse/api-sdk-node/dist/services/psc-verification-link/types";
+
+jest.mock("../../src/services/pscVerificationService");
+const mockCheckPlannedMaintenance = checkPlannedMaintenance as jest.Mock;
+
+const plannedMaintenanceResponseUp: ApiResponse<PlannedMaintenance> = {
+    httpStatusCode: 0,
+    resource: {
+        status: "UP",
+        message: "",
+        maintenance_start_time: new Date(),
+        maintenance_end_time: new Date()
+    }
+};
```

Comment:
Why not import PLANNED_MAINTENANCE from pscVerificationMock?


```suggestion
    resource: PLANNED_MAINTENANCE
};
```

Reply:
that's tidier, done.